CI/macos/build_ntv2.sh

```diff
@@ -16,10 +16,14 @@ _build_product() {
 
     step "Configure ("${ARCH}")..."
     cmake -S . -B build_${ARCH} -G Ninja ${CMAKE_CCACHE_OPTIONS} \
-        -DCMAKE_INSTALL_PREFIX="${BUILD_DIR}" \
+        -DCMAKE_INSTALL_PREFIX="${BUILD_DIR}\include" \
```

Comment:
You should not specify the `include` directory as a prefix. The prefix constitutes the "base" directory for installation, which you extend inside your CMake script by specifying either a custom subdirectory (`${CMAKE_INSTALL_PREFIX}/<YOUR_CUSTOM_DIR>`), or - which is preferred - the common directories set by `include(GNUInstallDirs)` (`${CMAKE_INSTALL_INCLUDEDIR}`, `${CMAKE_INSTALL_LIBDIR}`) which will be automatically composed based off the prefix directory.

Reply:
So the common approach is:

1. Set the prefix
2. Tell CMake the directory name(!) within the path specified by the prefix 

So if you run
```
install(TARGETS <YOUR_TARGET>
  RUNTIME DESTINATION bin
  LIBRARY DESTINATION lib
  ARCHIVE DESTINATION lib
  PUBLIC_HEADER DESTINATION include)
``` 
CMake will automatically install headers in `<PREFIX>/include`, libraries in `<PREFIX>/lib`, etc.